### ISSUE TITLE
Internet Explorer shows blank and freezes in autocomplete.

### ISSUE THREAD

**Issue by AnkushRoy** [Dogfalo/materialize#3816]
I am getting a blank div appearing in Internet Explorer when the autocomplete list is huge. For small lists it works fine.

Steps to reproduce:-
1. Check this codepen :- [codepenForBlankListInMaterializeAutocomplete](http://codepen.io/anon/pen/vXbaQo) in Internet Explorer (version 9/11).
2. Write A in the textbox.
3. Observe Apple/Apple00/Apple01....coming up in the list.
4. Choose Apple
5. Observe the blank list coming up. [Image](https://i.stack.imgur.com/uMAZD.png)

Code:- 
Head:
`<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=no" />
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="msapplication-tap-highlight" content="no" /><title>Test for  Autocomplete</title>
<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.7/css/materialize.min.css"/>`
HTML:
`<div class="row">
    <div class="col s12">
      <div class="row">
        <div class="input-field col s12">
          <i class="material-icons prefix">textsms</i>
          <input type="text" id="autocomplete-input" class="autocomplete">
          <label for="autocomplete-input">Autocomplete</label>
        </div>
      </div>
    </div>
  </div>`
Script:

```
$(function () {
  $('input.autocomplete').autocomplete({
        data: {
            "Apple": null,
            "Apple00": null,
            "Apple01": null,
            "Apple02": null,
            "Apple03": null,
            "Apple04": null,
            "Apple05": null,
            "Apple07": null,
            "Apple08": null,
            "Apple09": null,
            "Apple10": null,
            "Apple11": null,
            "Apple12": null,
            "Apple13": null,
            "Apple14": null,
            "Apple15": null,
            "Apple16": null,
            "Apple17": null,
            "Apple18": null,
            "Apple19": null,
            "Apple20": null,
            "Apple21": null,
            "Apple22": null,
            "Apple23": null,
            "Apple24": null,
            "Apple25": null,
            "Apple27": null,
            "Apple28": null,
            "Apple29": null,
            "Apple30": null,
            "Apple31": null,
            "Apple32": null,
            "Apple33": null,
            "Apple34": null,
            "Apple35": null,
            "Apple36": null,
            "Apple37": null,
            "Apple38": null,
            "Apple39": null,
            "Apple40": null,
            "Apple41": null,
            "Apple42": null,
            "Apple43": null,
            "Apple44": null,
            "Apple45": null,
            "Apple47": null,
            "Apple48": null,
            "Apple49": null,
            "Apple50": null,
            "Apple51": null,
            "Apple52": null,
            "Apple53": null,
            "Apple54": null,
            "Apple55": null,
            "Apple56": null,
            "Apple57": null,
            "Apple58": null,
            "Apple59": null,
            "Microsoft": null,
            "Google": 'http://placehold.it/250x250'
        }
    });

});
```

**Comment by fega** [comment]
Could you check if this is still a problem in the current release?

Closed due inactivity but feel free to reopen it if it is still necessary.

Thanks

**Comment by craigbroadman** [comment]
This is still an issue.

**Comment by craigbroadman** [comment]
I have tried adding this manually to materialize.js within the $.fn.autocomplete function but it doesn't seem to work. Any ideas?

**Comment by Dogfalo** [comment]
Fixed in df297b1e75070db64f330da1e789dc4c12f06d03
